content/en/docs/contribute/participating.md

```diff
@@ -103,6 +145,29 @@ For more about how to become a SIG Docs reviewer and the responsibilities and
 time commitment involved, see
 [Becoming a reviewer or approver](#becoming-an-approver-or-reviewer).
 
+#### Becoming a reviewer
+
+When you meet the
+[requirements for becoming a reviewer](https://github.com/kubernetes/community/blob/master/community-membership.md#reviewer),
+you can apply to become a SIG Docs reviewer. Even if you are already a reviewer
+in a different SIG, you still need to apply separately for SIG Docs.
+
+To apply, open a pull request to add yourself to the `reviewers` section of the
+[top-level OWNERS file](https://github.com/kubernetes/website/blob/master/OWNERS)
+in the `kubernetes/website` repository. Assign the PR to one or more current SIG
+Docs approvers.
+
+If your pull request is approved, you are now a SIG Docs reviewer, and
```

Comment:
```
If your pull request is approved, you are now a SIG Docs reviewer. [K8s-ci-robot] (https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) will assign and suggest you as a reviewer on new pull requests.

If you are approved, request that a current SIG Docs approver add you to the [@kubernetes/sig-docs-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews)
GitHub team. Only members of the `kubernetes-website-admins` GitHub team can add new members to a GitHub team.
```
> If your pull request is approved, you are now a SIG Docs reviewer. [K8s-ci-robot](https://github.com/kubernetes/test-infra/tree/master/prow#bots-home) will assign and suggest you as a reviewer on new pull requests.
> 
> If you are approved, request that a current SIG Docs approver add you to the [@kubernetes/sig-docs-pr-reviews](https://github.com/orgs/kubernetes/teams/sig-docs-pr-reviews)
GitHub team. Only members of the `kubernetes-website-admins` GitHub team can add new members to a GitHub team.